CHANGES

```diff
@@ -14,6 +14,14 @@ The rules for this file:
 
 ------------------------------------------------------------------------------
 
+2022-xx-xx  xiki-tempula, Henrik Jaeger
```

Comment:
@hejamu I mean here

Reply:
I have no issue with it :)

Reply:
I'd suggest to use the GitHub handle "hejamu" instead of clear names, simply because that's what we've been doing everywhere.

Reply:
```suggestion
2022-xx-xx  xiki-tempula, hejamu
```